src/node.ts

```diff
@@ -5,30 +5,59 @@ import { createNodeFileSystem } from "./vfs/createNodeFileSystem";
 import { build } from "./pipeline/build";
 import { consoleLogger } from "./logger";
 
-export async function run(args: {
-    configPath: string;
-    projectNames?: string[];
-}) {
+async function configForSingleFile(fileName: string): Promise<Config> {
+    return {
+        projects: [{
+            name: "main",
+            path: fileName,
+            output: process.cwd(),
+        }],
+        rootPath: process.cwd(),
+    };    
+}
```

Comment:
Neat! It would've been nice to enable debugging by default for convenience — `options: { debug: true }`